src/NMEA/EngineState.hpp

```diff
@@ -11,9 +11,12 @@
 */
 struct EngineState
 {
-  /* The engine box measured ignitions per second of the sparkplug. */
-  Validity ignitions_per_sec_available;
-  float ignitions_per_sec;
```

Comment:
As I wrote earlier, I'd prefer to have both fields (ignitions and revs) in the struct. Maybe one day the raw "ignitions" value is useful for something. Maybe we have some "engine on" InfoBox which works even if the user didn't configure an engine type. Whatever. But we can really spare those few bytes of RAM.

Reply:
This does not make any sens to me, because now I have to add dead code which is never used. 
You told me to calculate in the Device part the rpm from ign, this is where rpm data is put into NMEAInfo. Now I should add ignitions which is not used in the code?!?

Actually I thing it would be much more easy if you could implement some of this stuff by your own. Very time consuming this way. Implementation details I have to follow are only in your head...

Reply:
Please re-read https://github.com/XCSoar/XCSoar/pull/1201#issuecomment-1529423498 - yesterday, in that post, I told you to keep both. You don't have to "add dead code, you decided to remove that code, which already existed. The problem is not that implementation details are "in my head" - if I already told you, it's not just in my head, but also in yours, but you decided to ignore it.

Reply:
Yes, because it does not make any sens to me. RPMs are in the NMEAInfo struct. Now I have to add ignitions... your comment is on "Outdated" code?

Reply:
But no problemo, I add ignitions into NMEAInfo for future use...

Reply:
Yesterday, when I asked you to keep both, you didn't disagree. Sure, you didn't explicitly agree, but by silently doing something else, you make it harder than necessary to work with you. Because that's "only in your head"...

Reply:
Yea sorry, I did not see the point of keeping ignitions_per_second...but anyway, they are back